pyproject.toml

```diff
@@ -162,7 +162,7 @@ types-psutil = { version = "^5.8.13", optional = true }
 types-passlib = { version = "^1.7.7", optional = true }
 
 # mlstacks dependencies
-mlstacks = { version = "0.7.8", optional = true }
+mlstacks = { version = "0.7.11", optional = true }
```

Comment:
Seems like there is an error in the setup: `No matching distribution found for mlstacks==0.7.11`

Reply:
Yeah when I made the PR that version didn't exist (see the PR description). It should work now when / if I rerun.